pythonitalia/pycon
# video upload workflow as celery task

- Move video upload workflow to celery
- Fix tags, tests
- remove celery-heimdall
- tests
- improvements
- more tests
- changes
- check container build in ci
- docekrx
- test
- fix job
- remove debug
- fix ref
- Stop processing when the quota is finished
- more tests
- generator function
- remove temporal
- more tests
- assert build call
- remove never working action
- Revert "remove never working action"
- remove plan
- remove plan
- reduce sleep time
- dummy lock
- dummy lock
- changes
- changs
- changes
- fix
- redis
- ee
- parallel
- worker
- lock improvement
- Fix thumbnail upload

## Why



## How to test it
